src/utils/string/index.ts

```diff
@@ -32,3 +33,36 @@ export const omitTypename = (object) =>
 const DATE_FORMAT = "MMM d, yyyy, h:mm:ss aaaa";
 export const getDateCopy = (d: Date): string =>
   d ? format(new Date(d), DATE_FORMAT) : "";
+
+export const copyToClipboard = (str: string) => {
+  const el = document.createElement("textarea");
+  el.value = str;
+  document.body.appendChild(el);
+  el.select();
+  document.execCommand("copy");
+  document.body.removeChild(el);
+};
+
+export const sortFunctionString = (a, b, key) => {
```

Comment:
i think it makes sense to have some unit tests for `sortFunctionString` and `sortFunctionDate`

Reply:
i think unit tests are less important than ensuring the sorting functions do as they should in the context in which they are used. i don't mean don't write them when i say 'less important', i just mean that integration and e2e tests have higher value in asserting things work as they should.